src/AzureAuth/Program.cs

```diff
@@ -45,12 +49,20 @@ private static void Main(string[] args)
                 backend = TelemetryOutput.ApplicationInsights;
             }
 
+            var envVarsToCollect = new[]
+            {
+                Ado.Constants.SystemDefinitionId,
+                "QBUILD_DISTRIBUTED",

```

Comment:
We can move this to constants as they are `Cloud Build Agent` env var.

Reply:
Yeah I was still debating _which_ constants to put it in. It's not ADO, and it's not AzureAuth. This is a special snowflake of an env var and I think a constant on main might be the least bad thing.

Reply:
Or another constants file?

Reply:
Another constant file is fine too